api/v1/db/oracledb/LearningDB.sql

```diff
@@ -0,0 +1,192 @@
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (47, 8, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (48, 9, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (49, 10, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (50, 11, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (51, 12, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (52, 13, 4);
+
+CREATE TABLE ROUNDS (
+    ROUND_ID NUMBER NOT NULL,
+    ROUND VARCHAR2(255 BYTE),
+    PRIMARY KEY (ROUND_ID)
+);
+
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (1, 'blind');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (2, 'flop');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (3, 'turn');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (4, 'river');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (5, 'showdown');
+
+CREATE TABLE MEMBERS (
+    MEMBER_ID NUMBER NOT NULL,
+    MEMBER_NAME VARCHAR2(255 BYTE),
+    MEMBER_EMAIL VARCHAR2(255 BYTE),
+    MEMBER_LEVEL NUMBER NOT NULL,
+    MEMBER_EXP_OVER_LEVEL NUMBER NOT NULL,
+    MEMBER_PASSWORD VARCHAR2(255 BYTE),
+    PRIMARY KEY (MEMBER_ID),
+    UNIQUE (MEMBER_EMAIL)
+);
+
+CREATE TABLE GAMES (
+    GAME_ID NUMBER NOT NULL,
+    ROUND_ID NUMBER NOT NULL,
+    MINIMUM_BET NUMBER NOT NULL,
+    MAXIMUM_BET NUMBER NOT NULL,
+    BET_POOL NUMBER NOT NULL,
+    PRIMARY KEY (GAME_ID),
+    FOREIGN KEY (ROUND_ID) REFERENCES ROUNDS(ROUND_ID)
+);
+
+CREATE TABLE TABLE_CARDS (
+    OWNERSHIP_ID NUMBER NOT NULL,
+    GAME_ID NUMBER NOT NULL,
+    CARD_ID NUMBER NOT NULL,
+    PRIMARY KEY (OWNERSHIP_ID),
+    FOREIGN KEY (GAME_ID) REFERENCES GAMES(GAME_ID),
+    FOREIGN KEY (CARD_ID) REFERENCES CARDS(CARD_ID)
+);
+
+CREATE TABLE STATUSES (
+    STATUS_ID NUMBER NOT NULL,
+    STATUS VARCHAR2 (255 BYTE),
+    PRIMARY KEY (STATUS_ID)
+);
+
+INSERT INTO STATUSES (STATUS_ID, STATUS) VALUES (1,'folded');
+INSERT INTO STATUSES (STATUS_ID, STATUS) VALUES (2,'called');
+INSERT INTO STATUSES (STATUS_ID, STATUS) VALUES (3,'raised');
+INSERT INTO STATUSES (STATUS_ID, STATUS) VALUES (4,'checked');
+
+
+CREATE TABLE PLAYERS (
+    PLAYER_ID NUMBER NOT NULL,
+    MEMBER_ID NUMBER NOT NULL,
+    PLAYER_BET NUMBER NOT NULL,
+    STATUS_ID NUMBER NOT NULL,
+    PRIMARY KEY (PLAYER_ID),
+    FOREIGN KEY (MEMBER_ID) REFERENCES MEMBERS(MEMBER_ID),
+    FOREIGN KEY (STATUS_ID) REFERENCES STATUSES(STATUS_ID)
+);
+
+CREATE TABLE PLAYER_CARDS (
+    OWNERSHIP_ID NUMBER NOT NULL,
+    PLAYER_ID NUMBER NOT NULL,
+    CARD_ID NUMBER NOT NULL,
+    PRIMARY KEY (OWNERSHIP_ID),
+    FOREIGN KEY (PLAYER_ID) REFERENCES PLAYERS(PLAYER_ID),
+    FOREIGN KEY (CARD_ID) REFERENCES CARDS(CARD_ID)
+);
+
+CREATE TABLE PARTICIPATIONS (
+    PARTICIPATION_ID NUMBER NOT NULL,
+    MEMBER_ID NUMBER NOT NULL,
+    GAME_ID NUMBER NOT NULL,
+    PRIMARY KEY (PARTICIPATION_ID),
+    FOREIGN KEY (MEMBER_ID) REFERENCES MEMBERS(MEMBER_ID),
+    FOREIGN KEY (GAME_ID) REFERENCES GAMES(GAME_ID)
+);
```

Comment:
Is it necessary to have this table for keeping track of participations? I think you should be able to have a column on `PLAYERS` that references the game that they belong to since a player shouldn't be able to exist without a game